src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -114,13 +114,12 @@ public ConnectedOperations(MockMvc mvc,
 
     public void stubOutSamCalls(IamProviderInterface samService) throws Exception {
         Map<IamRole, String> snapshotPolicies = new HashMap<>();
-        snapshotPolicies.put(IamRole.CUSTODIAN, "hi@hi.com");
         snapshotPolicies.put(IamRole.STEWARD, "hi@hi.com");
         snapshotPolicies.put(IamRole.READER,  "hi@hi.com");
         Map<IamRole, String> datasetPolicies = new HashMap<>();
         datasetPolicies.put(IamRole.CUSTODIAN, "hi@hi.com");
         datasetPolicies.put(IamRole.STEWARD,  "hi@hi.com");
-        datasetPolicies.put(IamRole.INGESTER,  "hi@hi.com");
+        datasetPolicies.put(IamRole.SNAPSHOT_CREATOR,  "hi@hi.com");
```

Comment:
We probably want to switch the email to something obviously invalid, instead of using `hi.com`

Reply:
In case we want to do email validation, I think we do want to have these be valid, no? Also makes the test more readable I think.  We don't actually send anything with these...is that the concern?